.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
```

Comment:
Currently, with this PR, there would be a travis build (mvn test) : 
- for every master push
- for every branch pushed
- for every pull request

The uploading of the release package only happens when you explicitly go https://github.com/mesos/storm/releases and create a new release. A new tag would automatically be created. Travis would test that tag (mvn test) and if successful, start the "deploy" process. That is, basically, run ``bin/build_release.sh`` and upload the packages to the specific release page.